test/sign.test.js

```diff
@@ -0,0 +1,24 @@
+const { sign } = require("../lib");
+
+test("Sign of a negative number !=-0 should be -1", () => {
+    expect(sign(-123)).toBe(-1);
+});
+
+test("Sign of a postive number !=0 should be 1", () => {
+    expect(sign(123)).toBe(1);
+});
+
+test("Sign of a NaN should be NaN", () => {
+    expect(sign(NaN)).toBe(NaN);
+});
+
+test("Sign of a -0 should be -0", () => {
+    expect(sign(-0)).toBe(-0);
```

Comment:
I don't think this test will work or does it? Since 0 === -0, the `toBe(-0)` call will yield true for 0 and -0

Reply:
Maybe we could add another method `isNegativeZero` and `isPositiveZero` and use these in the tests and test them in their own tests :D

Reply:
![image](https://user-images.githubusercontent.com/14660993/31742888-2c878976-b427-11e7-9576-6e4f96390716.png)

Reply:
@BlakeGuilloud : Hahaha nice :)

@andreasgruenh: I looked at Math.sign and this function has those 5 return values I wrote test for. So let me do the following: 

* I fix the linting problems
* Remove both -0 and +0 tests 
* Write two new skeleton functions isNegativeZero and isPositiveZero for someone else to work on